utils/utils.hre.ts

```diff
@@ -23,12 +23,15 @@ export async function getSpokePoolDeploymentInfo(
   return { hubPool, hubChainId, spokeChainId };
 }
 
-export async function deployNewProxy(name: string, args: (number | string)[]): Promise<void> {
+type FnArgs = number | string;
+export async function deployNewProxy(name: string, constructorArgs: FnArgs[], initArgs: FnArgs[]): Promise<void> {
```

Comment:
Nice to name it as it is used in `run("verify:verify": ...);`.

```suggestion
export async function deployNewProxy(name: string, constructorArguments: FnArgs[], initArgs: FnArgs[]): Promise<void> {
```

Reply:
Fair but this is also used slightly further down as `constructorArgs` in the `deployProxy` fn. 

```
  const proxy = await upgrades.deployProxy(await getContractFactory(name, {}), initArgs, {
    kind: "uups",
    unsafeAllow: ["delegatecall"], // Remove after upgrading openzeppelin-contracts-upgradeable post v4.9.3.
    constructorArgs,
    initializer: "initialize",
  });
```

Reply:
Ah - that's annoying. Ignore this one then :+1: